boards/nordic/bm_nrf54l15dk/bm_nrf54l15dk_nrf54l05_cpuapp_s115_softdevice_defconfig

```diff
@@ -87,7 +87,6 @@ CONFIG_NRFX_SPIM30=y
 CONFIG_NRFX_SYSTICK=y
 CONFIG_NRFX_TEMP=y
 CONFIG_NRFX_TIMER00=y
-CONFIG_NRFX_TIMER10=y
```

Comment:
I think we should keep this entry here. If disabled, we should set it to `n` with a comment.